payments/models.py

```diff
@@ -145,9 +153,9 @@ class AdditionalProduct(AbstractBaseProduct):
 
     @property
     def product_type(self):
-        if self.service.is_fixed_service():
+        if self.service in ProductServiceType.FIXED_SERVICES():
             return AdditionalProductType.FIXED_SERVICE
-        elif self.service.is_optional_service():
+        elif self.service in ProductServiceType.OPTIONAL_SERVICES():
```

Comment:
I guess, these methods (`is_fixed_service` and `is_optional_service`) do not work anymore on an instance of `AdditionalProduct`? i.e. doing `self.service.is_fixed_service()` would return `None` or something? maybe we should get rid of those static methods then? they are used in some places that you did not update during this commit by the way ([example](https://github.com/City-of-Helsinki/berth-reservations/blob/c7d9fce97f38eba8181556834fccfe54e2ffd854/payments/schema/types.py#L151))

Reply:
ah, doing this `ProductServiceType(self.service).is_fixed_service()` works. I see. still, double check that we don't do `self.service.is_fixed_service()` anywhere

Reply:
Good catch, I forgot to check all places, but now I got rid of it, we can get around perfectly without it 👍